3-integration/connect-db/postgres/src/main/liberty/config/server.xml

```diff
@@ -32,7 +32,7 @@
             password="${db.password}"
             ssl="${db.ssl}" />
     </dataSource>
-    <variable name="db.ssl" defaultValue="true"/>
```

Comment:
Why we change default value to 'false'?

Reply:
Workaround ssl certificate restriction after upgrading postgres jdbc driver to 42.3.3 (see https://github.com/cloudcaptainsh/cloudcaptain/issues/234 for more info)